Libs/MRML/Widgets/qMRMLNodeComboBox.cxx

```diff
@@ -472,7 +476,7 @@ void qMRMLNodeComboBox::activateExtraItem(const QModelIndex& index)
       }
     }
   }
-  else if (d->RemoveEnabled && data.startsWith(tr("Delete current ")))
+  else if (d->RemoveEnabled && data.startsWith(tr("Delete current "))) // tr to delete ?
```

Comment:
All "startsWith" and "hasPostItem" should use data stored in `UserRole`, `UserRole+1`, ... (not the displayed `Qt::DisplayRole`).

ActionRole = Qt::UserRole -> "create"
NodeTypeRole = Qt::UserRole+1 -> "vtkMRMVolumeNode"